doc/man1/openssl-cmp.pod.in

```diff
@@ -858,6 +882,11 @@ Send request messages without CMP-level protection.
 File format to use when saving a certificate to a file.
 Default value is PEM.
 
+=item B<-crlform> I<PEM|DER>
+
+File format to use when saving a CRL to a file.
+Default value is DER.
```

Comment:
It should be documented why the default here is different from the certform default.